src/modules/image-upload/image-upload.js

```diff
@@ -72,31 +72,54 @@ function getSignature(policy) {
 export function getResponse(policyReq) {
 	const keyPrefix = getKeyPrefix(policyReq.auth.user, policyReq.uploadType, policyReq.textId),
 		policy = getPolicy(keyPrefix),
-		signature = getSignature(policy);
+		signature = getSignature(policy),
+		baseURL = 'https://up.bel.ng/',
+		filename = policyReq.filename.replace(/\s+/g, ' ');
+	let key = keyPrefix,
+		thumbnail = baseURL + keyPrefix.replace(/^uploaded/, 'generated'),
```

Comment:
baseURL is hard coded to up.bel.ng.
But wont the generated thumbnail be on s.bel.ng?

Unless, we have decided to not update the URL formats now.
In that case, is the lambda script updated?